ddapm_test_agent/agent.py

```diff
@@ -813,6 +826,8 @@ def main(args: Optional[List[str]] = None) -> None:
             "default snapshot directory %r does not exist or is not readable. Snapshotting will not work.",
             os.path.abspath(parsed_args.snapshot_dir),
         )
+    if parsed_args.disable_trace_parse_errors:
+        os.environ["DD_DISABLE_TRACE_PARSE_ERRORS"] = "True"
```

Comment:
🤔 why are we setting it in the env here?

Reply:
we should be passing it through `make_app` below to provide it to the tests

Reply:
should be set but lmk